### PR TITLE
Mention in the documentation that we ignore "kube-system" and "kube-public" namespaces for kube app discovery

### PR DESCRIPTION
**Pull request by AntonAM**
Improve Kube App Discovery documentation.

### CODE REVIEW

File: examples/chart/teleport-kube-agent/values.yaml [hunk 1]
```diff
@@ -384,6 +384,9 @@ databaseResources: []
 # documentation](../../../auto-discovery/kubernetes-applications/architecture.mdx)
 # for more details.
 #
+# NOTE: Discovery mechanism ignores Kubernetes services running in the "kube-system" and
+# "kube-public" namespaces.
+#
```
Comment (hugoShaka) [inline examples/chart/teleport-kube-agent/values.yaml:389]:
```suggestion
# <Admonition type="note">
# The Discovery mechanism ignores Kubernetes services running in the `kube-system` and
# `kube-public` namespaces.
# </Admonition>
```

**Review by hugoShaka — APPROVED** [review]
LGTM, you will need to re-render docs though with `make -C examples/chart render-chart-ref`

I'm not sure why the CI is not complaining.

**Review by marcoandredinis — APPROVED** [review]

**Comment by tigrato** [comment]
Now that we have a way to skip services, should we add a way to disable this skip for users that want Apps in those namespaces?

**Comment by AntonAM** [comment]
The customer who reported this didn't really need apps in those namespaces and agreed with our policy, but were just surprised because it was not mentioned in the docs. I think having this note should be fine for now, we can think about it if we know about clients who actually want it (there probably not much need for exposed apps in those namespaces)

**Pull request merged**